site/_resources/upgrading.md

```diff
@@ -12,18 +12,79 @@ This document describes the changes needed to upgrade your Contour installation.
 <br>
 
 <div class="alert-deprecation">
-<b>Deprecation Notice</b><br>
-The <code>IngressRoute</code> CRD has been deprecated and will be removed in Contour 1.6.
-Contour 1.5.1 continues to support the IngressRoute API, however we anticipate it will be removed in the future.
+<b>Removal Notice</b><br>
+The <code>IngressRoute</code> CRD has been removed in Contour 1.6.
```

Comment:
Also update the IngressRoute reference in `site/docs/master/annotations.md`?

Reply:
And delete `./site/docs/v1.6.0/ingressroute.md` and `./site/docs/master/ingressroute.md`?